pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
```

Comment:
As I mentioned on IRC. Yes. It is meant to allow comparing pushes.

On 11 December 2015 at 07:00, Tiramisu 1993 <notifications@github.com>
wrote:

> In pylib/mozhginfo/mozhginfo/push.py
> <https://github.com/mozilla/version-control-tools/pull/2#discussion_r47348211>
> :
>
> > @@ -0,0 +1,62 @@
> > +class Push(object):
>
> Hmm, I really haven't found what's cpm mean, the result google give me is
> https://github.com/alkant/cpm.
> BTW, do you mean *cmp* for cpm? @armenzg <https://github.com/armenzg> :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/version-control-tools/pull/2/files#r47348211>.
>



-- 
Zambrano Gasparnian, Armen
Automation & Tools Engineer
http://armenzg.blogspot.ca